src/container/index.tsx

```diff
@@ -24,6 +24,7 @@ export default function Container({
       disableContentPaddings={disableContentPaddings}
       {...externalProps}
       {...baseComponentProps}
+      __hiddenContent={disableContentPaddings && !props.children}
```

Comment:
is `disableContentPaddings` necessary? 
this should work regardless of padding, customers used this as a hack but I think it should work without it

Reply:
Makes sense! Will update it.